pandas/parser.pyx

```diff
@@ -662,7 +662,11 @@ cdef class TextReader:
 
         if isinstance(source, basestring):
             if not isinstance(source, bytes):
-                source = source.encode(sys.getfilesystemencoding() or 'utf-8')
+                if compat.PY36 and compat.is_platform_windows():
+                    fs_encoding = 'mbcs'
```

Comment:
Since `sys.getfilesystemencoding()` return the bad encoding (Utf-8) and is already called on `source`, I forced the encoding value.

Maybe not the best way to do (I don't know Pandas low-level machinery, maybe better way in the C code of the `new_file_source` function from `parser/io.h`), but AppVeyor say it work.